task11/TravelBuilder.java

```diff
@@ -0,0 +1,46 @@
+import java.util.ArrayList;
+
+/**
+ * Class that performs the travel along the route and prints out the needed statistics about the travel
+ */
+public class TravelBuilder {
```

Comment:
This not builder. Builder supposes to build something and return it to caller. This class doesn't build anything, more likely it estimates something